spring-grpc-spring-boot-autoconfigure/src/test/java/org/springframework/grpc/autoconfigure/common/GrpcCodecConfigurationTest.java

```diff
@@ -0,0 +1,82 @@
+package org.springframework.grpc.autoconfigure.common;
```

Comment:
Please add the pesky copyright :)

Reply:
done )

Reply:
There's a top-level file comment that we have to add too. Don't worry, we'll do it for you this time.